### ISSUE TITLE
Bitte den HmIP-FSI16 Unterputz Schaltaktor 16A mit Tastereingang implementieren

### ISSUE THREAD

**Issue by Sanguis1973** [rdmtc/RedMatic-HomeKit#298]
Meine Homebridge arbeitet sauber und fehlerfrei, einzig der oben genannte Aktor scheint nicht implementiert zu sein, denn egal wie oft ich die Bridge neu aufsetze und initialisiere, dieser eine Aktor wird als einziger niemals in der Homebridge gelistet

**Comment by Jemeins** [comment]
Wie der Zufall es will ist heute mein HmIP-FSI16 angekommen, ich kann das Problem bestätigen!

**Comment by itylux** [comment]
Man kann die FSI16 über einen Universalbutton und CCU Value hinzufügen. Allerdings übergibt der CCU Value dem Universalbutton nicht den Status wenn man das Licht über den realen Taster schaltet. Wenn jemand weiß wie man dem Universalbutton den aktuellen Status übergeben kann, wäre es ein Workaround. 

Bitte hier auch um implementation.

**Comment by itylux** [comment]
Hi, ich habe es über einen Flow gelöst. Hier findet ihr die Nodes:
https://github.com/rdmtc/RedMatic-HomeKit/issues/257#issuecomment-745560309

**Comment by Christchoper** [comment]
Hallo Zusammen,
ebenfalls bestätige ich das Issue. Wäre toll wenn der Aktor auch über den Homekit zur Verfügung stehen würde.
Danke im Voraus.

**Comment by MTDWOR** [comment]
Oh ja, das wäre schön, wenn der direkt verfügbar wäre.

**Comment by Christchoper** [comment]
Hallo Hobbyquaker,
Grossartig - Funktioniert eins-A.

Vielen Dank.

**Comment by MTDWOR** [comment]
Ja, der funktioniert problemlos. Issue kann geschlossen werden.

**Comment by hobbyquaker** [comment]
RedMatic-HomeKit **4.0.0** ist seit heute veröffentlicht – Installation in RedMatic 9 über den Node-RED-Palettenmanager (Suche nach `redmatic-homekit`), Release-Notes: https://github.com/rdmtc/RedMatic-HomeKit/releases/tag/v4.0.0.

HmIP-FSI16 ist als Schalter (Outlet/Lightbulb/… wählbar) abgebildet, der Tastereingang als opt-in programmierbarer Schalter.

Falls es mit 4.0.0 nicht wie erwartet funktioniert: bitte hier melden (Gerätetyp, Node-RED-Log mit Level „debug“ rund um den Start), dann wird das Issue wieder geöffnet.

---
*Written by Claude Fable on behalf of hobbyquaker.*
